Automattic/jetpack
# New connection banner for testing

Following brief conversation with @dereksmart, I put together this PR as a starting point for testing a new iteration of the connection banner.

The new version should ideally be split tested against the current one like @jeffgolenski did with the first iteration.

What I'm aiming for here is a simplification and hopefully more compelling explanation of Jetpack, building upon some of @jessefriedman's work with JPO.

What I've done in the PR is to create a new function called `render_banner_b()` within `class.jetpack-connection-banner.php` that, once the split testing logic is set up, can be called instead of the default `render_banner()` 50% of the time.

Testing instructions from @ebinnion:

-Checkout PR
- Deactivate Jetpack, and you should see banner on plugins page
- You should be able to go to `$site.com/wp-admin/options.php` and see the value for `jetpack_connection_banner_ab`. When the value is `1` you should see the previous banner. Then the value is `2`, you should see @richardmuscat's changes
- You can update the value to change which test you get, or you can delete the option and reload the page to test the split testing
